### ISSUE TITLE
Create Account and Spend problems?

### ISSUE THREAD

**Issue by Mandelhoff** [zack-bitcoin/amoveo#81]
I tried create account and spend from my full node account to another account and errors happened. I tried the public address with and without quotes (in base64 format).

![image](https://user-images.githubusercontent.com/35471470/35601444-b265862e-05f8-11e8-96c5-5d6c034fde22.png)

**Comment by zack-bitcoin** [comment]
The address should be an erlang binary. Like what you get from the command `api:pubkey().`
```
<<"This is an erlang binary">>.
```

**Comment by zack-bitcoin** [comment]
I am adding another example to the docs/api/commands_accounts.md file.

Thank you for this useful error report.

**Comment by Mandelhoff** [comment]
I tried with the updated format in the docs and no luck.
api:spend(<<"Base64Address">>, 60000).
I tried create_account too, but no luck with that.

![image](https://user-images.githubusercontent.com/35471470/35625124-c5108216-0656-11e8-9fe6-7d281028cc8a.png)


I also tried the erlang decimal format too, but no luck there:

![image](https://user-images.githubusercontent.com/35471470/35625151-d444a852-0656-11e8-83c7-cd4fe70a3c72.png)
